README.md

```diff
@@ -1,18 +1,26 @@
 # Building With Adobe DRM
 
+## Building the Application
+
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
 06. Symlink the "DRM_Connector_Prerelease" directory to "adobe-rmsdk" within the "Simplified-iOS" directory. (You will need to have obtained the Adobe DRM Connector prerelease from Adobe.)
-07. Follow the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf" to build OpenSSL (section 12.1) and cURL (section 12.3).*
+07. Build OpenSSL and cURL as described in the following "Building OpenSSL and cURL" section. Ensure you're in the "Simplified-iOS" directory before continuing to the next step.
 08. `sh adobe-rmsdk-build.sh`
 09. `cp APIKeys.swift.example Simplified/APIKeys.swift` and edit accordingly.
```

Comment:
I have a note saved: "for step 9, we need to explain where APIKeys.swift.example is to be coming from", so I'm guessing it didn't just appear in the file, I probably had to download it from a private repository?